src/test/java/com/squareup/kotlinpoet/FileSpecTest.kt

```diff
@@ -268,6 +268,44 @@ class FileSpecTest {
         |""".trimMargin())
   }
 
+  @Test fun singleImportEscapeKeywords() {
+    val source = FileSpec.builder("com.squareup.tacos", "Taco")
+            .addType(TypeSpec.classBuilder("Taco")
+                    .addProperty("madeFreshDate", ClassName("com.squareup.is.fun.in", "Date"))
+                    .build())
+            .build()
+    assertThat(source.toString()).isEqualTo("""
+        |package com.squareup.tacos
+        |
+        |import com.squareup.`is`.`fun`.`in`.Date
+        |
+        |class Taco {
+        |    val madeFreshDate: Date
+        |}
+        |""".trimMargin())
+  }
+
+  @Test fun doubleImportEscapeKeywords() {
```

Comment:
I don't feel like this test adds anything to the `singleImportEscapeKeywords()`, that one should be enough